v3-sdk/collecting-fees/src/example/Example.tsx

```diff
@@ -91,9 +97,20 @@ const Example = () => {
         )}
       <h3>{`Block Number: ${blockNumber + 1}`}</h3>
       <h3>{`Transaction State: ${txState}`}</h3>
-      <h3>{`${CurrentConfig.tokens.token0.symbol} Balance: ${tokenInBalance}`}</h3>
-      <h3>{`${CurrentConfig.tokens.token1.symbol} Balance: ${tokenOutBalance}`}</h3>
-      <h3>{`Position Ids: ${positionIds}`}</h3>
+      <h3>{`${CurrentConfig.tokens.token0.symbol} Balance: ${token0Balance}`}</h3>
+      <h3>{`${CurrentConfig.tokens.token1.symbol} Balance: ${token1Balance}`}</h3>
+      <div>
+        Positions:
+        {positionIds.length === positionsInfo.length &&
+          positionIds
+            .map((id, index) => [id, positionsInfo[index]])
+            .map((info) => {
```

Comment:
We could avoid the triple `map` and therefore the triple loop here by combining into 1. The same comment goes for the rest

Reply:
I prefer multiple map steps as it makes each step explicit, so want to leave that! Not really an efficiency concern when shaping data here + the very small size of the array